File: app/src/main/res/drawable/tripindirectory/adapters/PartnersViewHolder.java
package directory.tripin.com.tripindirectory.adapters;

import android.support.v7.widget.CardView;
import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageButton;
import android.widget.ImageView;
import android.widget.LinearLayout;
import android.widget.TextView;

import directory.tripin.com.tripindirectory.R;


public class PartnersViewHolder extends RecyclerView.ViewHolder {
    public LinearLayout mPartnerView;
    public CardView mCardView;
    public TextView mAddress;
    public ImageView mCall;
    public TextView mNatureOfBusiness;
    public TextView mTypeOfServices;
    public TextView mFleetInfo;
    public TextView mRouteInfo;
    public TextView mLastActiveTime;

    public TextView mCompany;

    public ImageView mStarView;
    public TextView mNumStarsView;
    public TextView mFleetSize;
    public ImageView mShareCompany;

    public PartnersViewHolder(View itemView) {
        super(itemView);
        mPartnerView = itemView.findViewById(R.id.partner_main_view);

        mCardView = itemView.findViewById(R.id.partner_card_view);
        mCall = itemView.findViewById(R.id.call);
        mAddress =  itemView.findViewById(R.id.company_address);
        mCompany = itemView.findViewById(R.id.company_name1);
        mNatureOfBusiness = itemView.findViewById(R.id.nature_of_business_info);
        mTypeOfServices = itemView.findViewById(R.id.type_of_service_info);
        mFleetInfo = itemView.findViewById(R.id.fleet_info);
        mRouteInfo = itemView.findViewById(R.id.route_info);
        mLastActiveTime = itemView.findViewById(R.id.last_active);

        mStarView =  itemView.findViewById(R.id.star);
        mNumStarsView = itemView.findViewById(R.id.post_num_stars);
        mFleetSize =  itemView.findViewById(R.id.fleet_size);
        mShareCompany =  itemView.findViewById(R.id.share_company);
    }
}

File: app/src/main/res/drawable/tripindirectory/newlookcode/PartnersViewHolder.kt
package directory.tripin.com.tripindirectory.newlookcode

import android.support.constraint.ConstraintLayout
import android.support.v7.widget.CardView
import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.ImageView
import android.widget.LinearLayout
import android.widget.TextView
import directory.tripin.com.tripindirectory.R

class PartnersViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var mCardView: CardView
    var mAddress: TextView
    var mFleetInfo: TextView
    var mCompany: TextView
    var mCall :TextView
    var mChat :TextView
    var mIsSelectedImg : ImageView
    var mOnlineStatus : ImageView
    var mThumbnail : ImageView
    var mChatParent : ConstraintLayout
    var mRateLayout : LinearLayout
    var mRatings : TextView
    var mReviews : TextView
    var mIsPromoted : TextView


    var isSelected : Boolean

    init {
        mThumbnail = itemView.findViewById(R.id.transporter_thumbnail)
        mCardView = itemView.findViewById(R.id.partner_card_view)
        mAddress = itemView.findViewById(R.id.company_address)
        mCompany = itemView.findViewById(R.id.company_name1)
        mFleetInfo = itemView.findViewById(R.id.fleet_info)
        mCall = itemView.findViewById(R.id.call)
        mChat = itemView.findViewById(R.id.chat)
        mIsSelectedImg = itemView.findViewById(R.id.ischecked)
        mChatParent = itemView.findViewById(R.id.chatconstrain)
        isSelected = false
        mOnlineStatus = itemView.findViewById(R.id.onlinestatus)
        mRateLayout = itemView.findViewById(R.id.ratings)
        mRatings = itemView.findViewById(R.id.rating)
        mReviews = itemView.findViewById(R.id.reviews)
        mIsPromoted = itemView.findViewById(R.id.promoted)



    }
}